### PR TITLE
Feature/mutations queries

### PR DESCRIPTION
**Pull request by kevinhartmann23**
## *Purpose:*
*What does this merge do? Check all that apply.*
- [x] Adds a new feature
- [x] Improves an existing feature
- [ ] Fixes a bug
- [ ] Cleans up (logs, tests, etc.)

## *Description: What does this PR do?*
*Briefly, but clearly describe what you did, and explain any new code.*
Successful mutation for useMutation hook!

## *Instructions for reviewer:*
*Where should the reviewer start?*
*How should this be manually tested?*
*Any background context you want to provide?*
*What are the relevant tickets?*

## *Issues:*
*Are there any concerns, issues, or bugs in this branch? If so describe them.*

## Checklist:
- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have used tests to prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my change

### CODE REVIEW

File: src/context/AppContext.js [hunk 1]
```diff
@@ -72,26 +71,13 @@ const AppProvider = ({children}) => {
...
-    //       endTime: "2021-04-22"
-    //     })
-    //     {
-    //       name
-    //     }
-    //  `
-  // const [ helperFunuction ] = useMutation(EVENTS_MUTATION)
+
+  //MUTATIONS
+  const [ mutateEvent, {error} ] = useMutation(EVENTS_MUTATION)
```
Comment (kristenmb) [inline src/context/AppContext.js:76]: We were so close yesterday! Thanks for muscling through this and figuring it out.

File: src/context/AppContext.js [hunk 2]
```diff
@@ -104,7 +90,18 @@ const AppProvider = ({children}) => {
 
   const updateEvents = (newEvent) => {
     setEvents([...events, newEvent])
-    // helperFunction({input: newEvent})
+    const { tourId, name, venueId, startTime, endTime } = newEvent
+    mutateEvent({
+      variables: {
```
Comment (kristenmb) [inline src/context/AppContext.js:95]: This makes perfect sense. I think if we hadn't already been working for hours yesterday we would have seen this.

**Review by kristenmb — APPROVED** [review]

File: src/components/form/Form.js [hunk 3]
```diff
@@ -40,13 +40,12 @@ const Form = () => {
...
-
     const newEvent = {
       "tourId": 1,
       "name": eventName,
-      "venueId": selectedVenue,
-      startTime,
-      endTime
+      "venueId": parseInt(selectedVenue),
+      "startTime": `${date}T${startTime}:00.000Z`,
+      "endTime": `${date}T${endTime}:00.000Z`
```
Comment (DrewBradley) [inline src/components/form/Form.js:48]: Word!

File: src/context/AppContext.js (hunk 1 above)
Comment (DrewBradley) [reply src/context/AppContext.js:76]: Whoa! useMutation? What is this sorcery? 😂

File: src/context/mutations.js [hunk 4]
```diff
@@ -2,12 +2,12 @@ import gql from 'graphql-tag';
...
-        startTime,
-        endTime
-      }
+    mutateEvent(input: $input) {
+      venueId,
+      name, 
+      tourId,
+      startTime, 
+      endTime
+    }
```
Comment (DrewBradley) [inline src/context/mutations.js:11]: Am I right in assuming that order matters here?

**Review by DrewBradley — APPROVED** [review]
Great job! I've got some questions about mutations, but we can discuss next time we meet.

File: src/context/AppContext.js (hunk 2 above)
Comment (richardltyler) [inline src/context/AppContext.js:95]: woot woot!  This is awesome!

**Review by richardltyler — APPROVED** [review]
This is so awesome!

**Pull request merged**